fdbcli/fdbcli.actor.cpp

```diff
@@ -989,6 +993,76 @@ void printStatus(StatusObjectReader statusObj, StatusClient::StatusLevel level,
 
 				if (statusObjConfig.get("log_routers", intVal))
 					outputString += format("\n  Desired Log Routers    - %d", intVal);
+
+				outputString += "\n  Usable Regions         - ";
+				if (statusObjConfig.get("usable_regions", intVal)) {
+					outputString += std::to_string(intVal);
+				} else {
+					outputString += "unknown";
+				}
+
+				StatusArray regions;
+				if (statusObjConfig.has("regions")) {
+					if (!activePrimaryDC.present()) {
+						outputString += "\n  WARNING: Unable to determine primary datacenter";
```

Comment:
If you put a string like this in the messages field, it should automatically get printed and you won't need this here.